### ISSUE TITLE
Is arm64 support a possibility?

### ISSUE THREAD

**Issue by 3rikk** [mandarons/yadd#5]
Hey
Firstly, this project looks really cool!

Like many though my "home server" consists entirely of a raspberry pi. Unfortunately there is no arm64 container available yet.

Thanks :)

**Comment by mandarons** [comment]
@3rikk Thank you for taking some time to submit this improvement opportunity. I have enabled ARM support and is now available on `main` branch. However, I don't have any ARM system to test the image on.

Can you please test it using `mandarons/yadd:main` tag on your ARM system? Please let me know if you find any issues. Else, I will include it in upcoming 1.1.0 release.

**Comment by 3rikk** [comment]
Thanks a lot :)
Unfortunately, the container gives out the following error at launch:

```
(node:8) UnhandledPromiseRejectionWarning: SequelizeConnectionError: SQLITE_CANTOPEN: unable to open database file at Database.<anonymous> (/app/node_modules/sequelize/lib/dialects/sqlite/connection-manager.js:70:34)

(Use `node --trace-warnings ...` to show where the warning was created)

(node:8) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)

(node:8) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
The webpage does load however you can't create any new services (the button doesn't do anything), probably due to the database being inaccessible

The folder ~/yadd/node-modules/ also doesn't exist, while ~/yadd/config/ does. 

The command I use for installation was:
`docker container run --name yadd -p 3334:3334 -v ${PWD}/config/:/app/config/ mandarons/yadd:main`
Other than that I didn't configure anything.

Is it a configuration mistake that I made? I apologise if that's the case

**Comment by mandarons** [comment]
@3rikk: Yeah, looks like `db.sqlite` creation is having some problem. Can you try the following:

1. In a new directory (e.g. `yadd`), create `config` folder.
2. Then run the command: `docker container run --name yadd -p 3334:3334 -v ${PWD}/config/:/app/config/ mandarons/yadd:main`

**Comment by 3rikk** [comment]
@mandarons hey there :)

Sorry for the late response. I tweaked your solution a bit and it worked!

Making _both_ the directories `config` and `node-modules` prior to running the container seems to make it all work.
I've tested the rest of the app a bit now too and everything seems to function properly.

**Comment by mandarons** [comment]
This feature will be released in 1.1.0.
